scripts/auto_dock.py

```diff
@@ -441,24 +592,49 @@ def openrover_linear_timer_cb(self, event):
         self.openrover_stop()
 
     def openrover_charging_cb(self, charging_msg):
-        self.is_docked = charging_msg.data
-        if self.docking_state == 'undock':
+        self.is_charging = charging_msg.data
+        self.docked_fun()
+
+    def in_range_service_call(self):
```

Comment:
This service call seems fine and the way you initialize the `in_range_service` proxy is fine as well. The only thing I would consider that should be changed is the naming. It is not super clear that "in range" means means charging